### ISSUE TITLE
Napostrophe

### ISSUE THREAD

**Issue by EbenSorkin** [source-foundry/glyph-filter-lists#14]
'napostrophe' - This glyphs was mistakenly added to an encoding (by IBM if I recall correctly) is now depreciated.

See: https://en.wikipedia.org/wiki/%C5%89

I recommend removing it.

**Comment by chrissimpkins** [comment]
Thanks Eben!  Here is where it shows up: https://github.com/source-foundry/glyph-filter-lists/search?q=napostrophe

Do you happen to know if it was removed from the MES specs, Underware data, or AGLFN yet?  I suppose that this might be ok for the Afrikaans writing system spec here, but I suggest that we keep the MES/AGLFN true to how they are defined.

**Comment by EbenSorkin** [comment]
I agree. In principle, it would not be useful to edit a list made by another group.
